scripts/error-codes/extractErrors.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+var babylon = require('babylon');
+var fs = require('fs');
+var glob = require('glob');
+var os = require('os');
+var path = require('path');
+var traverse = require('babel-traverse').default;
+
+var reactGlobPatterns = require('./globPatterns');
+
+var evalToString = require('./evalToString');
+var invertObject = require('./invertObject');
+var safeParseJSON = require('./safeParseJSON');
```

Comment:
As discussed in person, let's move this back inline – but move all the logic in this file into top-level functions and then just call it at the end of the file.